lib/components/artist/header.js

```diff
@@ -43,6 +45,15 @@ class Header extends React.Component {
     ARTemporaryAPIModule.setFollowStatus(!this.state.following, this.props.artist._id, (error, following) => {
       if (error) {
         __DEV__ ? console.error(error) : console.log(error);
+      } else {
+        Events.postEvent(this, {
+          name: following ? 'Follow artist' : 'Unfollow artist',
```

Comment:
Will we eventually have a place / list for all our events with the constants you think?